src/ArrayHelper.php

```diff
@@ -261,7 +284,7 @@ public static function getValue($array, $key, $default = null)
     }
 
     /**
-     * @param array|object $array array or object to extract value from
+     * @param mixed $array array or object to extract value from, otherwise method will return $default
```

Comment:
Why mixed?

Reply:
Because it will return $default if $array is not array and not object

Reply:
I think, if `$array` is not array and not object, should throws `InvalidArgumentException`.